src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -708,6 +714,12 @@
 
     public static final Key<Value<Vector3d>> RIGHT_ARM_ROTATION = null;
 
+    /**
+     * Represents the {@link Key} for representing whether something is right
+     * handed.
+     */
+    public static final Key<Value<Boolean>> RIGHT_HANDED = null;
```

Comment:
@ZephireNZ Encoding assumptions is unecessary, the structure in this PR is better than handedness being a boolean. Because if you assume that "not left hand" is equal to "right hand" then you instantly rule out any other hand. Maybe a mod lets you hold stuff in your tail or wear it on your head. It's no up to us to make that decision, we just need to be able to represent the possible states.